docker-for-windows/faqs.md

```diff
@@ -124,7 +124,7 @@ You can find a tutorial about running Windows containers on Windows Server in
 
 ### Can I install Docker Desktop on Windows 10 Home?
 
-Windows 10 Insider Preview (Windows 10 Home) users can now install [Docker Desktop Edge 2.2.2.0](https://download.docker.com/win/edge/43066/Docker%20Desktop%20Installer.exe) or a later release with the [experimental WSL 2 support](wsl-tech-preview.md). This requires Windows Insider Preview Build 19018 or later.
+Windows 10 Insider Preview (Windows 10 Home) users can now install [Docker Desktop Edge 2.2.2.0](https://download.docker.com/win/edge/43066/Docker%20Desktop%20Installer.exe) or a later release with the [experimental WSL 2 support](wsl.md). This requires Windows Insider Preview Build 19018 or later.
```

Comment:
this still mentions "insider preview" is that still needed now?

Reply:
Nice catch. I've add a new commit to update also the wording "Windows 10, version 2004" and also use our Stable release 2.3.0.2